public/index.pug

```diff
@@ -315,28 +321,8 @@ html(lang='en-US')
                   alt=`${icon.title} icon`
                 )
             .grid-item__row
-              if icon.guidelines
-                a.grid-item__link.link-button(
-                  title=`${icon.title} guidelines`,
-                  href=icon.guidelines,
-                  rel='noopener',
-                  target='_blank'
-                ) Brand Guidelines
-              if icon.license
-                a.grid-item__link.link-button.icon-legal(
-                  title=`${icon.title} icon license`,
-                  href=icon.license.url,
-                  rel='noopener',
-                  target='_blank'
-                ) #{ icon.license.type }
-              h2.grid-item__title #{ icon.title }
```

Comment:
Could you address it in this PR? It would be strange to remove useful functionality in the next release.

Reply:
No, this was removed on purpose. Look at my answer below. This PR is not to be committed just yet. The detail view PR will build upon this one, and that one will show license and guidelines. The information won't be lost after merging, since we will be merging the layout and detail view together.